.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,22 @@
+name: Docker Image CI
+on:
+  push:
+    branch: ['master', 'ci-tests']
```

Comment:
Maybe just build the image on code push to the master? Thoughts?

Reply:
Yes need to change, needed to be able to test. Trying to add in the call to kraken-hub now. Sorry about the preemptive PR opening

Reply:
No problem at all.

Reply:
Should be goo to review now. Need the kraken-hub PR: https://github.com/cloud-bulldozer/kraken-hub/pull/22/files to go in first to be able to correctly build this one